### ISSUE TITLE
Missing open brace for subscript

### ISSUE THREAD

**Issue by 01mf02** [Deducteam/Logipedia#28]
If we go for example to

http://logipedia.inria.fr/sttfa_bool__cases_term_6.def.html

then both "Body" and "Type_annotation" show:

    Missing open brace for subscript

**Comment by gabrielhdt** [comment]
This must be caused by the double `_` interpreted by the mathjax engine. Normally, the pretty printer [Logippedia](https://github.com/gabrielhdt/logippedia) should be run on these strings, which would escape the `_`.

In parallel, Gaspard has developed a LaTeX exporter that can be used instead of the pretty printer, so there are several solutions to a single problem.

**Comment by francoisthire** [comment]
It could be nice to have a fix rather quickly then. I think you should first chose one solution and discuss what is the best one after.
